src/config/webpack.base.ts

```diff
@@ -75,6 +75,9 @@ export default ({ mode, paths, env, sourceMaps }: IWebpackConfig): Configuration
   };
 
   return {
+    resolve: {
+      extensions: ['.ts', '.tsx', '.js']
```

Comment:
We dont use `jsx` but for typescript,  community wants us to use `tsx`.

[https://www.typescriptlang.org/docs/handbook/jsx.html](https://www.typescriptlang.org/docs/handbook/jsx.html)
[https://github.com/microsoft/TypeScript/issues/26489](https://github.com/microsoft/TypeScript/issues/26489)

[https://github.com/Microsoft/TypeScript/issues/30503](https://github.com/Microsoft/TypeScript/issues/30503)

Perhaps its better practice to use .jsx extensions instead of mixing JSX in .js files?

Reply:
@vietnogigoodrx .jsx seems to have been adopted as standard so fine by me